docs/configuration.md

```diff
@@ -10,6 +10,7 @@ export default function MyApp() {
   return (
     <>
       <Leva
+        detatched       // default = true, false would make the pane fill the parent dom node it's rendered in.
```

Comment:
The word is `detached`, is it only wrong here or also on the actual prop?

Reply:
Only here sorry about this. Oh and also on Discord.